configuration/pih/scripts/global/zl.js

```diff
@@ -222,15 +222,77 @@ function setupReturnVisitDateValidation(encounterDate, returnVisitDateLessThanEn
 
   jq(domEl).change(returnVisitDateValidation);
   beforeSubmit.push(returnVisitDateValidation);
-  
+
 
 
 }
 
 function restrictInputOnlyNumber(input_id) {
-  
+
   // Track the id of the input and change the type to number
-  jq(`#${input_id} input`).attr('type','number')
+  jq(`#${input_id} input`).attr('type', 'number')
+
+
+}
+
+
+/**
+ * Manages the activation of widget inputs based on the state of a radio button.
+ *
+ * @param {string|null} checkboxId - The ID of the checkbox associated with the radio button (optional).
+ * @param {string} radioButtonId - The ID of the radio button controlling the widget inputs.
+ * @param {string[]} widgetIds - The IDs of the widgets whose inputs should be enabled or disabled.
+ */
```

Comment:
Thanks for all the comments, makes it easier to quickly understand what is going on.